docs/src/man/hosting.md

```diff
@@ -282,6 +282,22 @@ See GitHub's manual for
 [Creating and using encrypted secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
 for more information.
 
+### Add code coverage from your deployed docs
+
+If you want code run during the documentation deployment to be covered by codecov,
+you can edit the end of the docs part of your workflow configuration file so that 
+`docs/make.jl` is run with the `--code-coverage=user` flag and the coverage reports
+are uploaded to codecov:
+
+```yaml
+      - run: julia --project=docs/ --code-coverage=user docs/make.jl
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+      - uses: julia-actions/julia-uploadcodecov@latest
```

Comment:
Two comments here:

1. It looks like [julia-uploadcodecov](https://github.com/julia-actions/julia-uploadcodecov) is deprecated in favour of [codecov/codecov-action](https://github.com/codecov/codecov-action)?
2. It might be better to recommend a specific (major) version, instead of `latest`?

Reply:
I made a new commit with these 2 points implemented (I think).